src/main/java/org/fundacionjala/coding/nestor/SpinWords.java

```diff
@@ -0,0 +1,32 @@
+package org.fundacionjala.coding.nestor;
+
+/**
+ * To Spinning word which length is greater than 4.
+ */
+public class SpinWords {
+  private static final String SPACE = " ";
+  private static final int FLAG = 4;
+
+  /**
+   * Method to return new String with spinner words.
+   *
+   * @param sentence type String.
+   * @return new string.
+   */
+  public String spinWords(final String sentence) {
+    String[] words = sentence.split(SPACE);
+    StringBuilder result = new StringBuilder();
+
+    for (String s : words) {
+      if (s.length() > FLAG) {
+        result.append(new StringBuilder(s).reverse().toString());
+      } else {
+        result.append(s);
+      }
+      result.append(SPACE);
+    }
+
+    String res = result.toString();
+    return res.substring(0, res.length() - SPACE.length());
```

Comment:
Instead of appending a blank space.
You can use a StringJoiner instead of StringBuilder.
Please take a look at this code.
```java
  public String spinWords(final String sentence) {
    String[] words = sentence.split(SPACE);
    StringJoiner result = new StringJoiner(SPACE);
    for (String s : words) {
      result.add(s.length() > FLAG ? new StringBuilder(s).reverse().toString() : s);
    }
    return result.toString();
  }
```